cmake/FindDAGMC.cmake

```diff
@@ -0,0 +1,17 @@
+# Try to find DAGMC
+#
+# Once done this will define
+#
+#  DAGMC_FOUND - system has DAGMC
+#  DAGMC_INCLUDE_DIRS - the DAGMC include directory
+#  DAGMC_LIBRARIES - Link these to use DAGMC
```

Comment:
Same comment as before about either listing all variables or none of them

Reply:
- [] Fix variable listing/descriptions in FindDAGMC.cmake